### ISSUE TITLE
[General Question] Is work still going on this?

### ISSUE THREAD

**Issue by AtomicCoding** [ihodev/sublime-da-ui#41]
The last commit was October 2nd, 2017.

**Comment by acheronfail** [comment]
@ihodev is active in the issues, and since this is going to be a paid package I believe most of the work will be done in larger chunks until it reaches the 1.0.0 milestone.

I'm reasonably sure that work is definitely going into this, but I can't speak for @ihodev! But he is usually open when things are not going to be developed anymore (look at the README for his other theme, Boxy https://github.com/ihodev/sublime-boxy).

**Comment by AtomicCoding** [comment]
@acheronfail Thanks!

**Comment by ihodev** [comment]
Yes, of course, there will be new releases. Currently I'm very busy due to changing my job position and the stack I learn to. `A File Icon` 4.0.0 is the next thing I want to finish when I get some free time. After that I'll continue to work on DA UI.

**Comment by swashata** [comment]
Hi @ihodev this may be a little out of line... But do you think you can cook something up for vscode? I really loved your theme and it would be really nice to have a pro theme for vscode 👍

**Comment by ihodev** [comment]
@swashata Yes, I have such plans 😉 DA UI for VS Code will be the next thing I want to implement after stable release of DA UI for ST.

**Comment by ChainsawBaby** [comment]
@ihodev just don't start neglecting ST. I've tried to moved to VS Code, it's just horrid IMO

**Comment by AtomicCoding** [comment]
Should I close this?

**Comment by lucsky** [comment]
We're now getting close to a year since last commit, just sayin'...

**Comment by KevinSjoberg** [comment]
@ihodev any update? I fully understand that you're busy and I truly appreciate all work you've put into DA UI. A quick update on where things are at and what to expect moving forward would be awesome. ❤️
